app/assets/stylesheets/modules/collection_details.css.scss

```diff
@@ -221,4 +256,11 @@ a.endpoint-label {
   width: 550px;
   padding: 0 15px 6px;
   font-size: .9em;
-}
\ No newline at end of file
+}
+
+.doi-badge {
```

Comment:
I really want the DOI badge to be a split badge. I think this looks much more pro...
![screen shot 2017-09-19 at 11 03 17 am](https://user-images.githubusercontent.com/7680119/30646420-954083d2-9dde-11e7-8957-6640f8071f78.png)